services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
+        last_name: str = "",
+) -> User:
+    return get_user_model().objects.create_user(
```

Comment:
You don’t need to `return` anything in create/update/delete methods, check everywhere